### ISSUE TITLE
[FR] Get Hermite data for mesh

### ISSUE THREAD

**Issue by trsh** [sceelix/Sceelix#13]
Would it be possible to get Hermite data instead of vertices and faces? So I can construct isosurface my self with Marching Cubes.

**Comment by trsh** [comment]
Ehh, I dont think so

**Comment by pabsilva** [comment]
Yeah, it definitely will. But that would be an interesting addition, but also a considerable enterprise.

I'd say it is something worth its own plugin/library, which anyone interested in following could work on rather independently (as part of this repo or a separate one).
